content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -245,9 +245,11 @@ init containers. [What's next](#what-s-next) contains a link to a more detailed
 
 ## Detailed behavior
 
-During the startup of a Pod, each init container starts following the order defined in the
-pod declaration, after the
-network and volumes are initialized. Each container must exit successfully before
+During the startup of a Pod, the kubelet waits until the networking and storage are 
+initialized, then starts each init container following the order in which
+these containers are defined in the Pod's `spec`.
```

Comment:
@pyaillet , This seems fine, though, what do you think about creating two sentences instead of one?
Something like:
During Pod startup, the kubelet waits to run init containers until the networking and storage are ready. Init containers start in the order they appear in the Pod's `spec`.

Reply:
Why not ! Would it be ok for you to keep `then` to show the sequence of these events though ?
Like:
During Pod startup, the kubelet waits to run init containers until the networking and storage are ready. Then init containers start in the order they appear in the Pod's spec.

Reply:
@pyaillet to me that sounds fine. You could write:
>  During Pod startup, the kubelet delays running init containers until the networking and storage are ready. Then the kubelet runs the Pod's init containers in the order they appear in the Pod's spec.

Reply:
Done ! Thank you :)